lib/puppet/functions/reboot/sleep.rb

```diff
@@ -0,0 +1,11 @@
+# Sleeps for specified number of seconds.
+Puppet::Functions.create_function(:'reboot::sleep') do
```

Comment:
Is the `sleep()` function not available from plans? Only curious, I trust there's good reason for this!

Reply:
I don't believe Puppet has a builtin `sleep` function. It's not a common need in catalogs.